README.md

```diff
@@ -86,64 +77,68 @@ set PATH=%JAVA_HOME%\bin;%PATH%
-mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence-rest.jar -DpomFile=%COHERENCE_HOME%\plugins\maven\com\oracle\coherence\coherence-rest\12.2.1\coherence-rest.12.2.1.pom
-mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence-http-grizzly.jar -DpomFile=%COHERENCE_HOME%\plugins\maven\com\oracle\coherence\coherence-http-grizzly\12.2.1\coherence-http-grizzly.12.2.1.pom
+mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence.jar -DpomFile=%COHERENCE_HOME%\plugins\maven\com\oracle\coherence\coherence\14.1.1\coherence.14.1.1.pom
+mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence-management.jar -DpomFile=%COHERENCE_HOME%\plugins\maven\com\oracle\coherence\coherence-management\14.1.1\coherence-management.14.1.1.pom
+mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence-rest.jar -DpomFile=%COHERENCE_HOME%\plugins\maven\com\oracle\coherence\coherence-rest\14.1.1\coherence-rest.14.1.1.pom
+mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence-http-netty.jar -DpomFile=%COHERENCE_HOME%\plugins\maven\com\oracle\coherence\coherence-http-netty\14.1.1\coherence-http-netty.14.1.1.pom
+mvn install:install-file -Dfile=%COHERENCE_HOME%\lib\coherence-jpa.jar -DpomFile=<coherence-jpa \(see below\)>.pom
 ```
 
-### Kubernetes Prerequisites
-
-To run the application in Kubernetes using the Coherence Operator, you must
-ensure the following requirements:
-
-* Ensure you have been able to install the Coherence Operator using the  [Quick Start Guide](https://oracle.github.io/coherence-operator/docs/2.0.0/#/about/03_quickstart).
-
-* Add the Helm repository. Execute the following commands to create a `coherence` helm repository:
-
-  ```bash
-   helm repo add coherence https://oracle.github.io/coherence-operator/charts
-   
-   helm repo update
-   ```
-
-### Get Coherence Docker Image
-
-Get the Coherence Docker image from the Oracle Container Registry:
-
-1. In a web browser, navigate to [Oracle Container Registry](https://container-registry.oracle.com) and click Sign In.
-2. Enter your Oracle credentials or create an account if you don't have one.
-3. Search for coherence in the Search Oracle Container Registry field.
-4. Click `coherence` in the search result list.
-5. On the Oracle Coherence page, select the language from the drop-down list and click **Continue**.
-6. Click **Accept** on the Oracle Standard Terms and Conditions page.
+For Coherence-JPA, use the following pom:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.oracle.toplink</groupId>
+  <artifactId>coherence-jpa</artifactId>
+  <version>14.1.1.0-0-0</version>
```

Comment:
This version looks like it has too many zeros, should be 14.1.1-0-0
Also I am not sure why we need this separate pom the pom and jar are available under a newly installed coherence home.

` find . -name 'coherence-jpa*'
./plugins/maven/com/oracle/toplink/coherence-jpa
./plugins/maven/com/oracle/toplink/coherence-jpa/14.1.1/coherence-jpa-14.1.1.pom
./plugins/maven/com/oracle/toplink/coherence-jpa/14.1.1/coherence-jpa-14.1.1.location
./lib/coherence-jpa.jar`

Reply:
Weird.  I wonder why it's not present in the dist file I was testing `coherence-java-14.1.1.0.0b77467.zip`.  I unzipped it and ran:  

```
<~/f/c/r/d/r/c/dist>-> find . -print | ag jpa
./coherence/lib/coherence-jpa.jar
<~/f/c/r/d/r/c/dist>->
```

Reply:
I installed via the installer.

Reply:
Okay, I'll update the instructions accordingly.